src/main/script/indexes.js

```diff
@@ -0,0 +1,11 @@
+db.MicroPostBean.createIndex({"created.timestamp":-1});
```

Comment:
Add comments to this script to motivate the various indices (see my original file)